PhysicsTools/NanoAOD/python/jetsAK4_CHS_cff.py

```diff
@@ -54,6 +54,18 @@
     tightJetIdLepVeto.filterParams, version = "RUN2UL16CHS"
 )
 
+run2_jme.toModify(
```

Comment:
This should then become `run2_jme_2016 | run2_jme_2017 | run_jme_2018`

Reply:
I changed it to `(run2_jme_2017 & run2_jme_2018).toModify(` since the `run2_jme_2016` modifier still lives above it on like 51.

Reply:
Indeed :+1: but shouldn't it be `|` instead of `&`? Perhaps both do the same...

Reply:
Makes sense, thanks, in that case it's the `|` we want

Reply:
Yes sorry about that. My thinking was mixed up. We do want the "or". Pushing after code checks momentarily.